pychromecast/controllers/homeassistant.py

```diff
@@ -69,7 +73,16 @@ def receive_message(self, _message, data: dict):
 
     def connect_hass(self, callback_function=None):
         """Connect to Home Assistant."""
+        if self.hass_connected:
+            callback_function()
```

Comment:
We should test `callback_function` is defined.
```suggestion
            if callback_function:
                callback_function()
```